doc/setup.rst

```diff
@@ -15,7 +15,18 @@ The following compilers are supported to compile both the library and the tests:
 Building from source
 --------------------
 
-First, clone this repository:
+First, install dependencies:
+
+* C/C++ compiler
+* Meson, Ninja
+* CMake (for subprojects)
+* pkg-config
+* libffi (libffi-dev)
```

Comment:
I'd probably recommend installing libgit2 since it'll likely be available in the OS's package manager.

Reply:
Added.